browser_use/llm/ollama/chat.py

```diff
@@ -23,6 +23,7 @@ class ChatOllama(BaseChatModel):
 	model: str
 
 	# # Model params
+	# TODO (matic): Why is this commented out?
```

Comment:
@gregpr07 I noticed that you commented this out. Is there something about `ollama` that doesn't permit using `temperature` parameter?